services/src/main/java/org/keycloak/services/resources/admin/RealmAdminResource.java

```diff
@@ -1369,7 +1369,7 @@ public KeyResource keys() {
     @Tag(name = KeycloakOpenAPI.Admin.Tags.REALMS_ADMIN)
     @Operation()
     @APIResponses(value = {
-        @APIResponse(responseCode = "200", description = "", content = @Content(schema = @Schema(implementation = String.class, type = SchemaType.ARRAY))),
+        @APIResponse(responseCode = "200", description = "OK"),
```

Comment:
@ahus1 here you have removed the content, why? In the rest of the endpoints it is always set